src/annotator/plugin/pdf.js

```diff
@@ -0,0 +1,123 @@
+import debounce from 'lodash.debounce';
+
+import * as pdfAnchoring from '../anchoring/pdf';
+import RenderingStates from '../pdfjs-rendering-states';
+// @ts-expect-error - Plugin module is CoffeeScript
+import Plugin from '../plugin';
+
+import PDFMetadata from './pdf-metadata';
+
+/**
+ * @typedef {import('../../types/annotator').Anchor} Anchor
+ * @typedef {import('../../types/annotator').Annotator} Annotator
+ * @typedef {import('../../types/annotator').HypothesisWindow} HypothesisWindow
+ */
+
+export default class PDF extends Plugin {
+  constructor(element, config) {
+    super(element, config);
+
+    /**
+     * The `Guest` instance for the current document.
+     * nb. This is initialized by `Guest` before it calls `pluginInit`.
+     *
+     * @type {Annotator}
+     */
+    this.annotator = /** @type {any} */ (null);
+
+    const window_ = /** @type {HypothesisWindow} */ (window);
+    this.pdfViewer = window_.PDFViewerApplication.pdfViewer;
+    this.pdfViewer.viewer.classList.add('has-transparent-text-layer');
```

Comment:
Is there any possibility of getting this this point and having `this.pdfViewer.viewer` be `undefined`?

Reply:
The previous code assumed that was not possible so I've kept the assumption here. From a look at the PDF.js source code, it appears to assume that `viewer` will always be set: https://github.com/mozilla/pdf.js/blob/6728c8fa6108fc0593ed7a60b90ed1b0a9f90af8/web/base_viewer.js#L145.